host/tests/L3_TestCases/dsDisplay/dsDisplayHelperClass.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, testName:str, qcId:str, log:logModule=None ):
 
         super().__init__(testName, qcId, log)
```

Comment:
Need to update test procedure doc for L3

Reply:
updated the test procedure  document for L3